apps/sms/views/inbox/js/inbox.js

```diff
@@ -974,7 +974,7 @@ var InboxView = {
     }
 
     if (thread.isDraft) {
-      this.removeThread(thread.id);
+      this.deleteThread(thread.id);
```

Comment:
It will make redundant call to the ```Notification.get```, but we do the same for the case when we just delete draft in selection mode. Maybe we can just file bug to improve this if we really want to.